tested/dsl/ast_translator.py

```diff
@@ -333,6 +336,21 @@ def _translate_to_ast(node: ast.Interactive, is_return: bool) -> Statement:
         return _convert_statement(statement_or_expression)
 
 
+def extract_comment(code: str) -> str:
```

Comment:
In the meeting we discussed passing the tokenized code. This way the code only has to be tokenized one time.

Did you look into this? And if so, why did you choose the current solution?

Reply:
Sadly something like that isn't possible from what I've seen.